src/locales/en/battle.json

```
@@ -32,6 +32,7 @@
   "attackMissed": "{{pokemonNameWithAffix}} avoided the attack!",
   "attackHitsCount": "Hit {{count}} time(s)!",
   "rewardGain": "You received\n{{modifierName}}!",
+  "rewardGainCount": "You received\n{{count}} {{modifierName}}!",
```

```suggestion
  "rewardGainCount": "You received\n{{count}} $t(modifierName, {\"count\": {{count}} })!",
```

This is necessary for plural handling of modifier name

e.g instead of `1 Berries` it can show `1 Berry` and `2 Berries`